samples/security/protection_psoc6/README.rst

```diff
@@ -0,0 +1,112 @@
+.. _prot-psoc6-sample:
+
+Sample secure domain application
+################################
+
+Overview
+********
+
+The :ref:`cy8ckit_062_wifi_bt` board has two core processors (Cortex-M4
+and Cortex-M0+). This sample application creates a demo with a protected
+area created on the Cortex-M0+ core (secure domain). On start CM0+ core
+protects its Flash, SRAM (using SMPU structures) from CM4 access and
+restricts access to some peripheral resources (using PPU structures).
+When CM4 tries to access protected SRAM area the system breaks it into
+a Hardware Fault exception.
+
+Requirements
+************
+
+- :ref:`cy8ckit_062_wifi_bt` board
+
+Building and Running
+********************
+
+#. Build the project
+
+.. code-block:: console
+
+   $ cd samples/security/protection_psoc6
+   $ mkdir build
+   $ cd build
+   $ cmake -G"Eclipse CDT4 - Ninja" ..
+   $ ninja
+
+#. Switch the DevKit into CMSIS-DAP mode using SW3 (LED2 should blink) and
+flash the board:
+
+.. code-block:: console
+
+   $<openocd_path>\bin\openocd -c "source [find interface/cmsis-dap.cfg]" \
+      -c "transport select swd" -c "source [find target/psoc6.cfg]" \
+      -c "if [catch {program {<zephyr_path>\samples\security\protection_psoc6\build\zephyr\zephyr.elf}} ] \
+         { echo {** Program operation failed **} } \
+         else { echo {** Program operation completed successfully **} }" \
+      -c "reset_config srst_only;reset run;psoc6.dap dpreg 0x04 0x00;shutdown"
+
+   $<openocd_path>/bin/openocd -c "source [find interface/cmsis-dap.cfg]" \
+      -c "transport select swd" -c "source [find target/psoc6.cfg]" \
+      -c "if [catch {program {<zephyr_path>\samples\security\protection_psoc6\build\prot_psoc6_unsecure-prefix\src\prot_psoc6_unsecure-build\zephyr\zephyr.elf}} ] \
+         { echo {** Program operation failed **} } \
+         else { echo {** Program operation completed successfully **} }" \
+      -c "reset_config srst_only;reset run;psoc6.dap dpreg 0x04 0x00;shutdown"
+
+
+#. Switch the DevKit back using SW3. Open two serial terminals (minicom, putty,
+etc.) and connect to the board with the following settings:
+
+- Speed: 115200
```

Comment:
Indent all these by three spaces too (so they're part of the numbered list item)